src/averageAges.js

```diff
@@ -14,12 +13,24 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+
+function calculateAvarageAge(people) {
+  const agesArr = people.map(person => person.died - person.born);
+  const avg = agesArr.reduce((prev, person) => {
+    return prev + person;
+  }, 0) / agesArr.length;
+
+  return avg;
+}
+
+function calculateMenAverageAge(people, century = false) {
+  let mens = people.filter(person => person.sex === 'm');
+
+  mens = century !== false
```

Comment:
I think you don`t need to check is century !== false and you don`t need to equalize it to 'false' in the function declaring